features/conversation/conversation-composer/conversation-composer-reply-preview.tsx

```diff
@@ -67,7 +54,7 @@ const Content = memo(function Content(props: {
   const topic = useCurrentConversationTopic();
 
   const { message: replyMessage } = useConversationMessageById({
-    messageId: replyingToMessageId,
+    messageId: replyingToMessageId!, // ! because we have enabled in the query
     topic,
   });
```

Comment:
But I put a comment saying that since we are checking enabled with react-query it should be fine? Do you recommend here to put `messageId: MessageId | undefined` instead? just felt weird doing that because for that function ideally I always want a messageId, passing undefined here wouldn't make sense.

export function useConversationMessageById({
  messageId,
  topic,
}: {
  messageId: MessageId;
  topic: ConversationTopic;
}) {
  const currentAccount = useCurrentAccount()!;
  const { data: messages } = useConversationMessages(currentAccount, topic);

  const cachedMessage = messages?.byId[messageId];

  // Only fetch the message if it's not already in the conversation messages
  const { data: message, isLoading: isLoadingMessage } = useQuery({
    ...getConversationMessageQueryOptions({
      account: currentAccount,
      messageId,
    }),
    enabled: !cachedMessage,
  });

  return {
    message: message ?? cachedMessage,
    isLoading: !cachedMessage && isLoadingMessage,
  };
}